<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>486</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Einstellungen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Visualisierung</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>502</width>
             <height>293</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Breite Simulationsfenster</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Höhe Simulationsfenster</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="canvas_width">
                <property name="minimum">
                 <number>100</number>
                </property>
                <property name="maximum">
                 <number>3000</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="canvas_height">
                <property name="minimum">
                 <number>50</number>
                </property>
                <property name="maximum">
                 <number>2000</number>
                </property>
                <property name="value">
                 <number>600</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QSpinBox" name="color_objects_r">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="color_objects_g">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="color_objects_b">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Pfeilfarbe</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QSpinBox" name="color_pointer_r">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="color_pointer_g">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="color_pointer_b">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="2">
               <widget class="QCheckBox" name="show_pointers">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diese Option aktiviert Pfeile, die auf die Körper in der Simulation zeigen (vor allem hilfreich bei grösseren Distanzen zwischen kleineren Körpern).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Pfeile anzeigen</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Objektfarbe</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="1">
               <widget class="QSpinBox" name="update_rate">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="t_factor">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Anzahl Berechnungen pro Sekunde&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Update-Rate (Geschwindigkeit) der Simulation</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="max_seconds">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Echtzeit-Sekunden bis zum Ende der Simulation. Eingabe 0 (null) ergibt eine endlose Simulation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>3600</number>
                </property>
                <property name="value">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_7">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;physikalisch: &amp;Delta;t&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Verschnellerungsfaktor (macht die Simulation ungenau)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Zeit in Sekunden bis zum Ende der Simulation</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="do_central_moving">
              <property name="toolTip">
               <string/>
              </property>
              <property name="toolTipDuration">
               <number>-1</number>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="accessibleDescription">
               <string/>
              </property>
              <property name="text">
               <string>beweglicher Zentralkörper</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QRadioButton" name="do_central_centered">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Der Zentralkörper wird bei Aktivieren dieses Feldes immer in der Mitte der Visualisierung angezeigt werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="toolTipDuration">
               <number>-1</number>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="accessibleDescription">
               <string/>
              </property>
              <property name="text">
               <string>zentrierter Zentralkörper</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Weiteres</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>502</width>
             <height>293</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="do_testing">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diese Option aktiviert möglicherweise unvollständige, noch in Entwicklung stehende Features.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Test-Features aktivieren</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="do_restart">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diese Option wird das Programm nach jeder Simulation neustarten. Somit wird eine weitere, unmittelbare Simulation möglich gemacht.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Neustart nach einer Simulation</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>203</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="b_close">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;schliessen ohne zu speichern&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Schliessen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_ok">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;speichern und schliessen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptionen">
    <property name="title">
     <string>Optionen</string>
    </property>
    <addaction name="actionVerlassen"/>
   </widget>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="actionSpeichern"/>
    <addaction name="actionSpeichern_unter"/>
    <addaction name="separator"/>
    <addaction name="actionLaden"/>
    <addaction name="actionLaden_von"/>
   </widget>
   <addaction name="menuOptionen"/>
   <addaction name="menuEinstellungen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVerlassen">
   <property name="text">
    <string>Verlassen</string>
   </property>
   <property name="toolTip">
    <string>Fenster schliessen ohne zu speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="toolTip">
    <string>Einstellungen speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSpeichern_unter">
   <property name="text">
    <string>Speichern unter...</string>
   </property>
   <property name="toolTip">
    <string>Einstellungen speichern unter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionLaden">
   <property name="text">
    <string>Laden</string>
   </property>
   <property name="toolTip">
    <string>Einstellungen laden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionLaden_von">
   <property name="text">
    <string>Laden von...</string>
   </property>
   <property name="toolTip">
    <string>Einstellungsdatei öffnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>canvas_width</tabstop>
  <tabstop>canvas_height</tabstop>
  <tabstop>color_objects_r</tabstop>
  <tabstop>color_objects_g</tabstop>
  <tabstop>color_objects_b</tabstop>
  <tabstop>color_pointer_r</tabstop>
  <tabstop>color_pointer_g</tabstop>
  <tabstop>color_pointer_b</tabstop>
  <tabstop>update_rate</tabstop>
  <tabstop>max_seconds</tabstop>
  <tabstop>do_testing</tabstop>
  <tabstop>do_restart</tabstop>
  <tabstop>b_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
